plugins/woocommerce-admin/client/customer-effort-score-tracks/product-mvp-ces-footer.tsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { closeSmall } from '@wordpress/icons';
+import { Pill } from '@woocommerce/components';
+import { CustomerFeedbackModal } from '@woocommerce/customer-effort-score';
+import { recordEvent } from '@woocommerce/tracks';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './product-mvp-ces-footer.scss';
+import { getStoreAgeInWeeks } from './utils';
+import {
+	ADMIN_INSTALL_TIMESTAMP_OPTION_NAME,
+	ALLOW_TRACKING_OPTION_NAME,
+	SHOWN_FOR_ACTIONS_OPTION_NAME,
+} from './constants';
+import { WooFooterItem } from '~/layout/footer';
+
+export const PRODUCT_MVP_CES_ACTION_OPTION_NAME =
+	'woocommerce_ces_product_mvp_ces_action';
+
+export const ProductMVPCESFooter: React.FC = () => {
+	const [ showFeedbackModal, setShowFeedbackModal ] = useState( false );
+	const { createSuccessNotice } = useDispatch( 'core/notices' );
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const {
+		storeAgeInWeeks,
+		cesAction,
+		allowTracking,
+		cesShownForActions,
+		resolving: isLoading,
+	} = useSelect( ( select ) => {
+		const { getOption, isResolving } = select( OPTIONS_STORE_NAME );
```

Comment:
Just a note here that `isResolving` will actually be `false` before it's `true`, which can cause some erratic behavior. I think it's more reliable to use `hasFinishedResolution` if that can serve your use case.